### PR TITLE
Adds the following items:

### PR DESCRIPTION
**Pull request by Bunnky**
Ramen Beef
Ramen Chicken
Ramen Seafood
Ramen Beef (Hearty)
Ramen Chicken (Hearty)
Ramen Seafood (Hearty)
Bagel
Bagel Bite (Special) (Needs More)
Bagel with Cream Cheese
Meatloaf
Sausage
Soylent
Tater Tots
Green Onion
Ketchup
Mustard
Cream Cheese
Pepper + Bush
Green Onion + Bush

### CODE REVIEW

**Comment by Sefiraat** [comment]
Yeah as you mentioned in Discord, your formatting's a bit whack - the content + the build looks valid at a glance. We will work through formatting on Discord until its resolved before I take a fuller look!

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 1]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+            CultivationStacks.SPINACH, CultivationStacks.PEA, Products.EGG.getFriedItem(),
+            new ItemStack(Material.CARROT), RAMEN_BEEF.getItem(), Products.BELL_PEPPER.getFriedItem(),
+            Products.GINGER.getChoppedItem(), Products.PARSLEY.getChoppedItem(), Products.GARLIC.getChoppedItem()
+        },
+        player ->
+            simplePlayerEffect(
+                player,
+                999,
+                new Pair<>(PotionEffectType.HEALTH_BOOST, 4),
+                new Pair<>(PotionEffectType.SPEED, 3)
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:511]:
```suggestion
                new Pair<>(PotionEffectType.HEALTH_BOOST, 1),
                new Pair<>(PotionEffectType.SPEED, 0)
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 2]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+            Products.ONION.getChoppedItem(), SlimefunItems.BUTTER, Products.RED_MUSHROOM.getChoppedItem(),
+            SlimefunItems.SALT, RAMEN_CHICKEN.getItem(), Products.CHILLI_PEPPER.getSlicedItem(),
+            Products.GINGER.getChoppedItem(), Products.PARSLEY.getChoppedItem(), Products.GARLIC.getChoppedItem()
+        },
+        player ->
+            simplePlayerEffect(
+                player,
+                999,
+                new Pair<>(PotionEffectType.HEALTH_BOOST, 4),
+                new Pair<>(PotionEffectType.SPEED, 3)
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:528]:
```suggestion
                new Pair<>(PotionEffectType.HEALTH_BOOST, 1),
                new Pair<>(PotionEffectType.SPEED, 0)
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java [hunk 3]
```diff
@@ -3401,5 +3465,149 @@ private CultivationStacks() {
...
+    public static final SlimefunItemStack RAMEN_CHICKEN_HEARTY = Theme.themedSlimefunItemStack(
+        "CLT_RAMEN_CHICKEN_HEARTY",
+        Skins.RAMEN_HEARTY.getPlayerHead(),
+        CultivationThemes.FOOD,
+        "Ramen (Chicken) (Hearty)",
+        "A filling choice.",
+        "",
+        Theme.CLICK_INFO.asTitle("Healing", "Full"),
+        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 5"),
+        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 4")
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java:3519]:
```suggestion
        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 2"),
        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 1")
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java [hunk 4]
```diff
@@ -3401,5 +3465,149 @@ private CultivationStacks() {
...
+    public static final SlimefunItemStack RAMEN_SEAFOOD_HEARTY = Theme.themedSlimefunItemStack(
+        "CLT_RAMEN_SEAFOOD_HEARTY",
+        Skins.RAMEN_HEARTY.getPlayerHead(),
+        CultivationThemes.FOOD,
+        "Ramen (Seafood) (Hearty)",
+        "A satisfying choice.",
+        "",
+        Theme.CLICK_INFO.asTitle("Healing", "Full"),
+        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 5"),
+        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 4")
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java:3531]:
```suggestion
        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 2"),
        Theme.CLICK_INFO.asTitle("Effect 2", "Speed + 1")
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 5]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+            Products.EGG.getBoiledItem(), Products.PUFFERFISH.getGrilledItem(), CultivationStacks.CURRY_LEAF,
+            Products.LIME.getSlicedItem(), RAMEN_SEAFOOD.getItem(), Products.SALMON.getGrilledItem(),
+            Products.GINGER.getChoppedItem(), Products.CILANTRO.getChoppedItem(), Products.GARLIC.getChoppedItem()
+        },
+        player ->
+            simplePlayerEffect(
+                player,
+                999,
+                new Pair<>(PotionEffectType.HEALTH_BOOST, 4),
+                new Pair<>(PotionEffectType.SPEED, 3)
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:545]:
```suggestion
                new Pair<>(PotionEffectType.HEALTH_BOOST, 1),
                new Pair<>(PotionEffectType.SPEED, 0)
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 6]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+        CultivationStacks.BAGEL_CREAM_CHEESE,
+        RecipeTypes.FINISHING,
+        new ItemStack[]{
+            null, BAGEL.getItem(), null,
+            null, Ingredients.CREAM_CHEESE.getItem(), null,
+            null, null, null
+        },
+        player -> simplePlayerEffect(player, 10,
+                                     new Pair<>(PotionEffectType.HEALTH_BOOST, 0)
+        )
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:587]:
```suggestion
        player -> simplePlayerEffect(player, 10, new Pair<>(PotionEffectType.HEALTH_BOOST, 0))
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 7]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+        new ItemStack[]{
+            Ingredients.KETCHUP.getItem(), new ItemStack(Material.EGG), Products.BREAD.getChoppedItem(),
+            new ItemStack(Material.MILK_BUCKET), Products.BEEF.getGroundItem(), Products.THYME.getChoppedItem(),
+            SlimefunItems.SALT, Products.ONION.getChoppedItem(), Ingredients.MUSTARD.getItem()
+        },
+        player -> simplePlayerEffect(player, 999,
+                                     new Pair<>(PotionEffectType.HEALTH_BOOST, 5),
+                                     new Pair<>(PotionEffectType.DAMAGE_RESISTANCE, 4),
+                                     new Pair<>(PotionEffectType.FAST_DIGGING, 3),
+                                     new Pair<>(PotionEffectType.SLOW, 2)
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:602]:
```suggestion
                                     new Pair<>(PotionEffectType.HEALTH_BOOST, 3),
                                     new Pair<>(PotionEffectType.DAMAGE_RESISTANCE, 2),
                                     new Pair<>(PotionEffectType.FAST_DIGGING, 1),
                                     new Pair<>(PotionEffectType.SLOW, 1)
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java [hunk 8]
```diff
@@ -3401,5 +3465,149 @@ private CultivationStacks() {
...
+        Skins.MEATLOAF.getPlayerHead(),
+        CultivationThemes.FOOD,
+        "Meatloaf",
+        "Nothing like it! Let's go.",
+        "",
+        Theme.CLICK_INFO.asTitle("Healing", "Full"),
+        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 6"),
+        Theme.CLICK_INFO.asTitle("Effect 2", "Damage Reduction + 5"),
+        Theme.CLICK_INFO.asTitle("Effect 3", "Haste + 4"),
+        Theme.CLICK_INFO.asTitle("Effect 4", "Slowness + 3")
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/CultivationStacks.java:3576]:
```suggestion
        Theme.CLICK_INFO.asTitle("Healing", "Full"),
        Theme.CLICK_INFO.asTitle("Effect 1", "Health Boost + 4"),
        Theme.CLICK_INFO.asTitle("Effect 2", "Damage Reduction + 3"),
        Theme.CLICK_INFO.asTitle("Effect 3", "Haste + 2"),
        Theme.CLICK_INFO.asTitle("Effect 4", "Slowness + 2")
```

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 9]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+
+    public static final Food SOYLENT = new Food(
+        CultivationStacks.SOYLENT,
+        RecipeTypes.FINISHING,
+        new ItemStack[]{
+            Products.BEEF.getBlendedItem(), Products.BANANA.getBlendedItem(), Products.COCOA.getBlendedItem(),
+            Products.AVOCADO.getBlendedItem(), new ItemStack(Material.WATER_BUCKET), Products.VANILLA.getBlendedItem(),
+            Products.SPINACH.getBlendedItem(), Products.SOY_BEANS.getBlendedItem(), new ItemStack(Material.ICE)
+        },
+        player -> simplePlayerEffect(player, 0,
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:625]: Bring player, 0 and each pair onto their own lines

File: src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java [hunk 10]
```diff
@@ -459,6 +461,186 @@ null, new ItemStack(Material.WITHER_ROSE), null,
...
+        CultivationStacks.TATER_TOTS,
+        RecipeTypes.FINISHING,
+        new ItemStack[]{
+            null, SlimefunItems.SALT, null,
+            null, Products.POTATO.getFriedItem(), null,
+            null, null, null
+        },
+        player -> simplePlayerEffect(player, 1,
+                                     new Pair<>(PotionEffectType.SPEED, 1)
+        )
```
Comment (Sefiraat) [inline src/main/java/dev/sefiraat/cultivation/implementation/slimefun/items/Foods.java:641]:
```suggestion
        player -> simplePlayerEffect(player, 1, new Pair<>(PotionEffectType.SPEED, 1))
```

**Comment by Bunnky** [comment]
Not sure what I did up there xD however I have included the suggestions, removed an unused Skin, and added an Import for ParticleUtils. I tried to do everything through the IDE because I thought I could, but I couldn't figure it out so ignore the goofs lol :)

**Comment by Bunnky** [comment]
@J3fftw1

**Review by Sefiraat — APPROVED** [review]
Looking good to me!

**Comment by Sefiraat** [comment]
> @J3fftw1 

That's not how you request a review, I'll send you some screenshots when I'm at my PC, I've added Jeff for ya :)

File: src/main/java/dev/sefiraat/cultivation/api/utils/Skins.java [hunk 11]
```diff
@@ -507,7 +507,31 @@ public enum Skins {
...
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/54977-meatloaf
+    MEATLOAF("98eeafdf6d7c6c9943e44bfcfdbe037d6c15d6d5e354c3ccf72bf886cde4e1c6"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/59814-johnsonville-italian-sausage
+    SAUSAGE("49d6afe4e80fd6a70e4833852062f6d5a2abf0da007fbf38b31fe4048b3cf9bd"),
+    // https://minecraft-heads.com/custom-heads/plants/6358-potato
+    TATER_TOT("dbfb6d4a54c17e2748437acc7098fbb1a3a12a407f51b3e49542332714846fd8"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/8748-bagel
+    BAGEL("502e92f13de3bee69228c384478e761230681e5fce9bda195daeaf8484139331"),
+    // https://minecraft-heads.com/custom-heads/food%20&%20drinks/8750-poppyseed-bagel
+    BAGEL_EVERYTHING("653aaae67dda225b78e82886d046c67fdf37f1bbe2dfc55f1324781f8159f5"),
```
Comment (J3fftw1) [inline src/main/java/dev/sefiraat/cultivation/api/utils/Skins.java:530]: Bagel everything isn’t that descriptive

**Review by J3fftw1 — CHANGES_REQUESTED** [review]

File: src/main/java/dev/sefiraat/cultivation/api/utils/Skins.java (hunk 11 above)
Comment (Sefiraat) [reply src/main/java/dev/sefiraat/cultivation/api/utils/Skins.java:530]: Everything bagel is, annoyingly, the name of a type of bagel, and marking up with the archetype first followed by the descriptor is in line with my code style, so while it seems to not make sense it annoyingly does. everything Bagel, 'murica

**Pull request merged**